html/food.js

```diff
@@ -2237,5 +2241,64 @@ export const food = {
 		sanity: sanity_supertiny,
 		stack: stack_size_smallitem,
 		mode: 'together'
+	},
+	
+	// DST Return of Them: Waterlogged beta
+	fig: {
+		name: 'Fig',
+		isfruit: true,
+		fruit: 0.5,
+		health: 0,
+		hunger: calories_small,
+		perish: perish_fast,
+		sanity: 0,
+		stack: stack_size_smallitem
+	},
+	fig_cooked: {
+		name: 'Cooked Fig',
+		isfruit: true,
+		fruit: 0.5,
+		precook: 1,
+		health: healing_tiny,
+		hunger: calories_medsmall,
+		perish: perish_superfast,
+		sanity: 0,
+		stack: stack_size_smallitem
+	},
+	batnose: {
+		name: 'Naked Nostrils',
+		ismeat: true,
+		meat: 0.5,
+		health: healing_small,
+		hunger: calories_small,
+		sanity: -sanity_small,
+		cook: 'batnose_cooked',
+		//idk why the guide uses "morsel" 'cause the prefab name is "smallmeat"
```

Comment:
If I remember correctly from starting this 10 years ago, all of the names in the code were lifted from game data, so maybe they changed it at some point or it was based on something else. We could switch to `smallmeat` at some point to get better parity, but that doesn't have to be part of this PR.